keen/__init__.py

```diff
@@ -90,17 +91,21 @@ def count(event_collection, timeframe=None, timezone=None, interval=None, filter
     example: [{"property_name":"device", "operator":"eq", "property_value":"iPhone"}]
     :param group_by: string or array of strings, the name(s) of the properties you would
     like to group you results by.  example: "customer.id" or ["browser","operating_system"]
+    :param order_by: dictionary object containing the property_name to order by and the
+    desired direction of sorting. Example: {"property_name":"result", "direction":"DESC"}
+    :param limit: positive integer limiting the displayed results of a query using order_by
```

Comment:
Should this comment that's copy/pasted everywhere mention that `order_by` can only be there if `group_by` is not `None` and not empty? Similarly, should it indicate that `limit` only applies if there is an `order_by` param provided too? Should those three params really be an instance of some new `GroupByOptions` class that upon creation validates those constraints instead? You already chose to pack the `direction` option into the dict for `order_by` so at the least the `limit` option could be right there in that dict so it's clear that it belongs with `order_by`, no?

Really in general the fact that adding a single param means copy/pasting this same stuff all over place maybe indicates that this parameter list really at some point should have been moved to an `Options` property bag instead, maybe? That'd be a bigger, super breaking change, though.

Reply:
This actually mimics the API itself. I thought it was odd too.  I don't want to diverge too far from the cURL syntax. I will, however, update the comments with the group_by necessity.